src/index.ts

```diff
@@ -17,9 +17,53 @@
 
 import * as m from 'mithril';
 import App from './components/App';
+import './styles/global.scss';
+import BarlowRegularWoff from '../fonts/BarlowRegular.woff';
+import BarlowRegularWoff2 from '../fonts/BarlowRegular.woff2';
+import BarlowSemiBoldWoff from '../fonts/BarlowSemiBold.woff';
+import BarlowSemiBoldWoff2 from '../fonts/BarlowSemiBold.woff2';
 
-m.mount(document.body, App);
+// These should be identical to the @font-face rules in styles/_fonts.scss. Both
+// are included so there is a CSS fallback in case loading via JS fails for some
+// reason. Sadly, we can't just use the same family 'Barlow' for both of these
+// as we can in CSS (and toggle the font used just with the font-weight
+// property), as FontFace overwrites on loading the second one (unlike CSS,
+// which supports multiple @font-face rules for a single family).
+const barlowRegularFont = new FontFace(
+    'Barlow Regular',
+    `local('Barlow'), url('${BarlowRegularWoff2}') format('woff2'), url('${BarlowRegularWoff}') format('woff')`,
+    { style: 'normal', weight: '400' },
+) as FontFace;
```

Comment:
I don't _think_ the type assertion is required here, as when I removed this, typescript didn't complain. Ditto for line 41 below.